profiler/src/ProfilerEngine/Datadog.Profiler.Native/RuntimeIdStore.cpp

```diff
@@ -19,25 +19,17 @@ Error("unknown platform");
 
 const char* const RuntimeIdStore::ServiceName = "RuntimeID Store";
 const char* const RuntimeIdStore::ExternalFunctionName = "GetRuntimeId";
-const char* const RuntimeIdStore::NativeLoaderFilename =
-#ifdef _WINDOWS
-#ifdef BIT64
-    "Datadog.AutoInstrumentation.NativeLoader.x64" LIBRARY_FILE_EXTENSION;
-#else
-    "Datadog.AutoInstrumentation.NativeLoader.x86" LIBRARY_FILE_EXTENSION;
-#endif
-#else
-    "Datadog.Trace.ClrProfiler.Native.so";
-#endif
+const char* const RuntimeIdStore::NativeLoaderFilename = "Datadog.Trace.ClrProfiler.Native" LIBRARY_FILE_EXTENSION;
 
 bool RuntimeIdStore::Start()
 {
+    // the native loader is always available in the same directory
```

Comment:
For Windows MSI installs, this is currently not the case for 64-bit MSI installations that are instrumenting 32-bit applications. One way we could solve this is to create a symlink during the installation, though I'm not sure if there are additional permissions needed to make it work. Or, we could install ALL of the `win-x86` assets in the `Program Files (x86)` directory. What do you think?

BTW here's a SO link for creating a symlink during MSI install: https://stackoverflow.com/a/51793730

Reply:
> One way we could solve this is to create a symlink during the installation

Other alternatives:
- hard link (what are the pros/cons vs symlink?)
- does msi allow installing the same file twice in different locations?

> additional permissions needed to make it work

The msi already requires admin permissions, so that should be enough?

Reply:
If we can avoid symlinks/hardlinks I'd really rather not, as they can cause weird issues when removing the file (e.g. we have to do this in the Nuke build, and recursive deleting blows up, so we explicitly remove the hard links first)

> Does msi allow installing the same file twice in different locations

I don't see why not, though would double the size of the MSI

> The msi already requires admin permissions, so that should be enough?
I agree

Reply:
> Does msi allow installing the same file twice in different locations?
> I don't see why not, though would double the size of the MSI.

I think we can include a file in the msi once and copy it into multiple places during installation. The mapping of inputs and outputs is not 1:1.

Reply:
Resolving this thread as we've decided to go with the approach [here](https://github.com/DataDog/dd-trace-dotnet/pull/3060#discussion_r940674918) instead